spyder/widgets/helperwidgets.py

```diff
@@ -455,5 +459,44 @@ def test_msgcheckbox():
     box.exec_()
 
 
+class PanelEmptyWidget(QWidget):
+    """Update progress installation widget."""
+
+    def __init__(self, parent, icon_filename, text, description):
+        super().__init__(parent)
+        # Setup attributes
+        self.icon_filename = icon_filename
+        self.text = text
+        self.description = description
+
+        # Setup layout
+        panel_empty_layout = QVBoxLayout()
+        self.label_empty = QLabel()
+        self.label_empty.setText(self.text)
+        self.label_empty.setAlignment(Qt.AlignCenter)
+        self.label_empty.setStyleSheet(
+            f"font-size: {DialogStyle.TitleFontSize}")
+        self.label_empty2 = QLabel()
+        self.label_empty2.setText(self.description)
+        self.label_empty2.setAlignment(Qt.AlignCenter)
+        self.label_empty2.setStyleSheet(
+            f"font-size: 10pt;background-color:{SpyderPalette.COLOR_OCCURRENCE_3}")
+
+        # Image
+        image_path = get_image_path(self.icon_filename)
+        image = QPixmap(image_path)
+        self.image_label = QLabel()
+        image_height = int(image.height())
+        image_width = int(image.width())
+        image = image.scaled(image_width, image_height, Qt.KeepAspectRatio,
+                             Qt.SmoothTransformation)
+        self.image_label.setPixmap(image)
+        self.image_label.setAlignment(Qt.AlignCenter)
+        panel_empty_layout.addWidget(self.image_label)
+        panel_empty_layout.addWidget(self.label_empty)
+        panel_empty_layout.addWidget(self.label_empty2)
+
+        self.setLayout(panel_empty_layout)
```

Comment:
Checking @ccordoba12 comment about adding a border and to further improve this class definition. We could define it like this:

```suggestion
class PanelEmptyWidget(QFrame):
    """Widget to show a panel/plugin functionality description."""

    def __init__(self, parent, icon_filename, text, description):
        super().__init__(parent)
        # Setup widgets
        # Image
        image_path = get_image_path(icon_filename)
        image = QPixmap(image_path)
        image_label = QLabel(self)
        image_height = int(image.height())
        image_width = int(image.width())
        image = image.scaled(image_width, image_height, Qt.KeepAspectRatio,
                             Qt.SmoothTransformation)
        image_label.setPixmap(image)
        image_label.setAlignment(Qt.AlignCenter)
        image_label.setStyleSheet("border: 0px")

        # Main text
        text_label = QLabel(text, parent=self)
        text_label.setAlignment(Qt.AlignCenter)
        text_label.setWordWrap(True)
        text_label_qss = qstylizer.style.StyleSheet()
        text_label_qss.QLabel.setValues(
            fontSize=DialogStyle.TitleFontSize,
            border="0px"
        )
        text_label.setStyleSheet(text_label_qss.toString())

        # Description text
        description_label = QLabel(description, parent=self)
        description_label.setAlignment(Qt.AlignCenter)
        description_label.setWordWrap(True)
        description_label_qss = qstylizer.style.StyleSheet()
        description_label_qss.QLabel.setValues(
            fontSize="10pt",
            backgroundColor=SpyderPalette.COLOR_OCCURRENCE_3,
            border="0px",
            padding="20px"
        )
        description_label.setStyleSheet(description_label_qss.toString())

        # Setup layout
        panel_empty_layout = QVBoxLayout()
        panel_empty_layout.addWidget(image_label)
        panel_empty_layout.addWidget(text_label)
        panel_empty_layout.addWidget(description_label)
        panel_empty_layout.setContentsMargins(0, 0, 0, 0)
        self.setLayout(panel_empty_layout)

        # Setup border style
        self.setFocusPolicy(Qt.StrongFocus)
        self._apply_stylesheet(False)

    def focusInEvent(self, event):
        self._apply_stylesheet(True)
        super().focusOutEvent(event)

    def focusOutEvent(self, event):
        self._apply_stylesheet(False)
        super().focusOutEvent(event)

    def _apply_stylesheet(self, focus):
        if focus:
            border_color = QStylePalette.COLOR_ACCENT_3
        else:
            border_color = QStylePalette.COLOR_BACKGROUND_4

        qss = qstylizer.style.StyleSheet()
        qss.QFrame.setValues(
            border=f'1px solid {border_color}',
            margin='0px 1px 0px 1px',
            padding='0px 0px 1px 0px',
            borderRadius='3px'
        )

        self.setStyleSheet(qss.toString())
```

With that change things look something like this:

![empty](https://user-images.githubusercontent.com/16781833/236584996-482a013c-6312-443a-83f2-d3963182982a.gif)

This suggestion adds the usage of `qstylizer` to manage the widgets stylesheets, changes the parent class from `QWidget` to `QFrame` (to enable adding the border style), changes the naming of the widgets and organization of the class `__init__` code, and also sets `wordWrap`  for the labels to enable the widget being resized.

Reply:
This looks pretty nice, thanks @dalthviz! I have an additional suggestion according to your gif above: I think we should leave a bit of vertical space (2px?) between the combobox in the second row of the Find plugin and PanelEmptyWidget. Otherwise, the two widgets look to close to each other.